.github/workflows/weekly-serverless.yml

```diff
@@ -0,0 +1,34 @@
+name: Test Serverless Environment
+run-name: Creating Serverless Environment by @${{ github.actor }}
+
+on:
+    workflow_dispatch: # TODO: remove
+    schedule:
+      - cron: '0 0 * * 1' # every Monday at 00:00
+
+jobs:
+  naming:
+    runs-on: ubuntu-latest
+    outputs:
+      deployment_name: ${{ steps.set_deployment_name.outputs.date-name }}
+    steps:
+      - name: Set deployment name
+        id: set_deployment_name
+        run: |
+          echo "::set-output name=date-name::weekly-env-$(date +'%d-%b')"
```

Comment:
`set-output` is [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)
Usage example:
```
deployment = "weekly-env-$(date +'%d-%b')"
echo "date-name=$deployment" >> GITHUB_OUTPUT
```